lib/features/modeling/cmd/ParticipantMultiplicityHandler.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Command handler that fires on
+ */
+import { is } from 'bpmn-js/lib/util/ModelUtil';
+import { updateBands } from '../../../util/BandUtil';
+
+export default function ParticipantMultiplicityHandler(eventBus, bpmnFactory, elementRegistry) {
+  this._eventBus = eventBus;
+  this._bpmnFactory = bpmnFactory;
+  this._elementRegistry = elementRegistry;
+}
+
+ParticipantMultiplicityHandler.prototype.updateAllBands = function(participant) {
+  let activities = this._elementRegistry.filter(
+    element => is(element, 'bpmn:ChoreographyActivity') &&
+      element.bandShapes.some(band => band.businessObject.id === participant.businessObject.id)
+  );
+  activities.forEach(activity => updateBands(activity, this._eventBus));
+};
+
+ParticipantMultiplicityHandler.$inject = [
+  'eventBus',
+  'bpmnFactory',
+  'elementRegistry'
+];
+
+ParticipantMultiplicityHandler.prototype.execute = function(context) {
+  if (context.hasMultiplicity) {
+    delete context.participant.businessObject.participantMultiplicity;
+  } else {
+    context.participant.businessObject.participantMultiplicity = this._bpmnFactory.create(
```

Comment:
Put `context.participant.businessObject` in a local variable so the line isn't that long.